### ISSUE TITLE
Please remove "bgp_local_pref" for BGP Confederation peers

### ISSUE THREAD

**Issue by netfreak98** [natesales/pathvector#86]
Would it be possible to remove the following part from the generated peer config when BGP confederation is activated?

`bgp_local_pref = {{ $peer.LocalPref }}; # pathvector:localpref
`

Otherwise it would override the local-pref to 100 if the confederation ASN is not the same like the local configured one.

Example where it works:

```
  AS49697 CONFED AS65002:
    local-asn: 65002
    asn: 65002
    confederation: 49697
    confederation-member: true
    template: confed
    multihop: true
    listen4: 185.121.69.145
    listen6: 2001:678:a00:1000::d
    neighbors:
      - "185.121.69.144"
      - "2001:678:a00:1000::c"
```

Example where it **not** works:

```
  AS49697 CONFED AS65001:
    local-asn: 65002
    asn: 65001
    confederation: 49697
    confederation-member: true
    template: confed
    multihop: true
    listen4: 185.121.69.145
    listen6: 2001:678:a00:1000::d
    neighbors:
      - "185.121.69.150"
      - "2001:678:a00:1000::b"

root@c02:~# birdc show route 8.8.8.0/24 all
BIRD 2.0.8 ready.
Table master4:
[..]
                     unicast [AS49697_CONFED_AS65001v4 2021-12-14 21:07:56 from 185.121.69.150] (100/?) [AS15169i]
	via 185.121.69.164 on vlan1000
	Type: BGP univ
	BGP.origin: IGP
	BGP.as_path: (65001) 15169
	BGP.next_hop: 185.121.69.150
	BGP.med: 0
	BGP.local_pref: 100
	BGP.community: (49697,3000)
```

After removing the generated "BGP.local_pref:" part from the peer config it starts to behave like having a normal IBGP session with the same ASN configured: 

```
root@c01:~# birdc show route 8.8.8.0/24 all
BIRD 2.0.8 ready.
Table master4:
8.8.8.0/24           unicast [AS49697_CONFED_AS65002v4 2021-12-14 22:57:17 from 185.121.69.144] * (100/?) [AS15169i]
	via 185.121.69.161 on vlan1000
	Type: BGP univ
	BGP.origin: IGP
	BGP.as_path: (65002) 15169
	BGP.next_hop: 185.121.69.144
	BGP.med: 0
	BGP.local_pref: 211
	BGP.community: (49697,3000)
```

**Comment by netfreak98** [comment]
As a addition, it's also important to remove "remove-private-as" from the peer config, otherwise it would lead to loops ^^

Beyond that things are working well

```
root@e01:~# birdc show route for 85.190.148.140 protocol AS49697_CONFED_AS65002v4 all
BIRD 2.0.8 ready.
Table master4:
85.190.148.0/24      unicast [AS49697_CONFED_AS65002v4 2021-12-15 08:15:45 from 185.121.69.144] * (100/?) [AS199610i]
	via 185.121.69.134 on ens19
	Type: BGP univ
	BGP.origin: IGP
	BGP.as_path: (65002 65001) 199610
	BGP.next_hop: 185.121.69.144
	BGP.local_pref: 210
	BGP.community: (49697,3000) (65101,1082) (65102,1000) (65103,276) (65104,150) (65534,2000)
	BGP.large_community: (6695, 1000, 1)

```

**Comment by natesales** [comment]
Thanks for the report! I've just released [5.8.0](https://github.com/natesales/pathvector/releases/tag/v5.8.0) including the new `set-local-pref` boolean option. Set this to false and you should be good to go.
